finetrainers/trainer.py

```diff
@@ -845,17 +873,19 @@ def validate(self, step: int) -> None:
                     artifact_value = wandb.Image(filename)
                 elif artifact_type == "video":
                     logger.debug(f"Saving video to {filename}")
-                    export_to_video(artifact_value, filename, fps=15)
+                    # TODO: this should be configurable here as well as in validation runs where we call the pipeline that has `fps`.
+                    export_to_video(artifact_value, filename, fps=15) 
```

Comment:
Let's make this configurable via CLI. I would call it `save_fps` or similar and default to 15 or 30